go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/scothis/riff-buildpack-group
```

Comment:
I'm not sure I follow what this is for?
Is there some go code that is being compiled for the builder itself (I know the ITs is a separate module)

Reply:
It shouldn't be there.